docs/tempo/website/setup/linux.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Deploy on Linux
+menuTitle: Deploy on Linux
+description: Learn how to deploy Tempo on Linux
+weight: 100
+draft: true
+---
+
+<!-- This page was migrated from GET and needs to have the repo and file locations updated for Tempo. -->
+
+# Deploy on Linux
+
+This guide provides a step-by-step process for installing Tempo on Linux.
+It assumes you have access to a Linux machine and the permissions required to deploy a service with network and filesystem access.
+At the end of this guide, you will have deployed a single Tempo instance on a single node.
+
+## Before you begin
+
+To follow this guide, you need:
+
+- A running Grafana instance (see [installation instructions](https://grafana.com/docs/grafana/latest/setup-grafana/installation/))
+- An Amazon S3 compatible object store
+<!-- - Git and Docker installed to run the TNS app -->
+
+### System requirements
+
+This configuration is an example you can use as a starting point.
+You may need to have more resources for your system than the minimum specifications listed below.
+Additional adjustments will be necessary for a production environment.
+
+You must have the permissions required to deploy a service with a network and file system access.
+
+Your Linux system should have at least:
+
+- 4 CPUs
+- 16 GB of memory
+
+## Set up an object storage bucket
+
+Tempo uses object storage as the backend for its trace storage.
+It also uses object storage for storing various administrative credentials and data related to the state of the system.
```

Comment:
I think `administrative credentials` aren't in Tempo, but mentioning other data is good.

Reply:
Updated